src/sockperf.cpp

```diff
@@ -1731,10 +1737,19 @@ static int resolve_sockaddr(const char *host, const char *port, int sock_type,
     int res;
     struct addrinfo *result;
 
+#ifdef __windows__
+    WSAData wsaData;
+    if ((res = WSAStartup(MAKEWORD(2, 2), &wsaData)) != 0) {
+        log_err("WSAStartup failed %d\n", res);
+        res = SOCKPERF_ERR_SOCKET;
+        return res;
+    }
+
+#endif
     if (host != NULL) {
         std::string path = host;
         struct sockaddr_store_t *tmp = ((sockaddr_store_t *)addr);
-        if (path.size() > 0 && path[0] == '/') {
+        if (path.size() > 0 && path[0] == os_directory_path) {
```

Comment:
Could you please add an example how AF_UNIX socket name looks like on Windows in command line and in feedfile?

Reply:
Where should I add the example?

Reply:
As in previous PR, please add the example to readme file and to comment for `set_sockets_from_feedfile`

Reply:
@agalanin-at-nvidia 
In unix it should be: /tmp/test
in windows it should be: \tmp\test 

btw, I am now wondering what happens later with comm_channel patch. 
From what I checked, PATH must be the same, so in case we have windows host to linux BF we have problem with this structure.

Reply:
1. Have you updated the regex for AF_UNIX sockets?
2. How to specify a socket on drive `D:` for example?

Reply:
Ok that's very interesting! 

1. It works with prefix `\` but only for main directory 
If I want `D:` e.g I need to omit the prefix `\`

2. It also works with `/` prefix on windows
See attached 

![windows no prefix needed3](https://user-images.githubusercontent.com/44922030/183897062-b8e76eba-394a-4c34-a977-2a0612aface6.JPG)
![windows no prefix needed1](https://user-images.githubusercontent.com/44922030/183897068-8d7f7d0b-5cc1-42da-8cca-99dd7a582f34.JPG)
![windows no prefix needed2](https://user-images.githubusercontent.com/44922030/183897069-88fa53b7-f92c-4066-89a3-489fff4cb615.JPG)

Reply:
So I guess we don't need prefix at all for windows, furthermore- if we use prefix it won't work in case we want to change directory, e.g `D:`